shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -358,6 +366,14 @@ object GlutenConfig {
       .filter(_._1.startsWith(backendPrefix))
       .foreach(entry => nativeConfMap.put(entry._1, entry._2))
 
+    // Pass the latest tokens to native
+    nativeConfMap.put(
+      GLUTEN_UGI_TOKENS,
+      UserGroupInformation.getCurrentUser.getTokens.asScala
+        .map(_.encodeToUrlString)
+        .mkString("\0"))
+    nativeConfMap.put(GLUTEN_UGI_USERNAME, UserGroupInformation.getCurrentUser.getUserName)
```

Comment:
if Kerberos is not enabled, then the username will be `none` ?

Reply:
It will be local linux username.